tests/test_notification_system.py

```
@@ -0,0 +1,251 @@
+            name="Test Email",
+            type=NotifierType.EMAIL,
+            address="test@example.com",
+            enabled=True,
+            subscribed_events=["SCAN_COMPLETED"],
+            smtp_host="smtp.example.com",
+            smtp_port=587,
+            smtp_user="testuser",
+        )
+
+        # Set password via property (should encrypt)
+        test_password = "my_secret_password"
```

Ruff (0.14.13)

Possible hardcoded password assigned to: "test_password"

(S105)